src/App/common/Navigation/Search/index.js

```diff
@@ -20,6 +20,13 @@ export const Search = () => {
     );
   };
 
+  useEffect(() => {
+    const queryParam = new URLSearchParams(location.search).get("query");
+    if (queryParam !== null) {
+      setSearchValue(queryParam);
+    }
+  }, [location.search]);
+
```

Comment:
Nice idea, but I don't think it fully solves the discussed issue, since the URL is still not being synced with the `searchValue` state. So to make it work as desired, it's unavoidable to connect those two 👌

Reply:
done :-)

Reply:
Works much better now, but there are some possible improvements. For example when we change the value in the URL it's not immediately synced with the search value, which is not a huge issue, but I'd say for me it'd give much better effect if those two would be just the same value at any time. But it's just optional 😎